docs/targets/googlecloudstorage.md

```diff
@@ -0,0 +1,46 @@
+# Event Target for GoogleCloudStorage 
+This event target integrates with GoogleCloudStorage, using received Cloud Event messages to create Blob items.
+
+## Prerequisites
+
+* A storage bucket is expected to exist prior to running this event target. If you haven't already done so, create a bucket by following the instructions from the [Cloud Storage How-To Guides](https://cloud.google.com/storage/docs/how-to).
```

Comment:
Let's follow the same template as sources. I would like to see these prerequisites as an unordered list. Then you can tell the user how to find them by adding some sections about the prerequisite. Example:

`## Prerequisites
- storage bucket
- service account

### Storgae Bucker

A storage bucket is expected to exist prior to running this event target. If you haven't already done so, create a bucket by following the instructions from the [Cloud Storage How-To Guides](https://cloud.google.com/storage/docs/how-to).
`